config/translate.go

```diff
@@ -89,11 +89,11 @@ func TranslateFromV1(old v1.Config) types.Config {
 				Node: types.Node{
 					Filesystem: filesystem.Name,
 					Path:       string(oldFile.Path),
-					Mode:       int(oldFile.Mode),
 					User:       types.NodeUser{ID: oldFile.Uid},
 					Group:      types.NodeGroup{ID: oldFile.Gid},
 				},
 				FileEmbedded1: types.FileEmbedded1{
+					Mode: int(oldFile.Mode),
```

Comment:
All of these schema changes should be a separate commit (or part of the previous).